website/source/docs/agent/options.html.md

```diff
@@ -853,6 +853,15 @@ default will automatically work with some tooling.
         contents of each entry.
 
 
+* <a name="auto_encrypt"></a><a href="#auto_encrypt">`auto_encrypt`</a>
+    This object allows setting options for the AutoEncrypt feature.
```

Comment:
Are we going to always refer to this as the "AutoEncrypt" feature? If so, I'll need to make updates in the guide.

Reply:
Do you have another suggestion? I forgot what we say in the guide.

Reply:
```suggestion
    This object allows setting options for the `auto_encrypt` feature.
```